src/test/java/org/broadinstitute/hellbender/utils/runtime/StreamingProcessControllerUnitTest.java

```diff
@@ -190,13 +98,13 @@ public void testStderrRedirect() throws TimeoutException {
     }
 
     @Test(timeOut = 10000)
-    public void testFIFOLifetime() throws TimeoutException {
+    public void testFIFOLifetime() {
```

Comment:
Does this test really test the lifetime of a fifo?  It looks like it just makes sure that it's opened and closed properly.

Reply:
The FIFO's lifetime is dictated by the lifetime of the controller; this test validates that bye checking that the FIFO doesn't survive termination of the controller. Maybe there is a better name ?